src/feed/index.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import Event from './event.js'
+import _ from 'lodash'
+
+class Comments extends React.Component {
+
+  static propTypes = {
+    records: React.PropTypes.array,
+    empty: React.PropTypes.string.isRequired
+  }
+
+  static defaultProps = {
+    empty: "No records found."
+  }
+
+  render() {
+    if(_.isEmpty(this.props.records)) {
```

Comment:
Another place to use LoadingContainer. Existing components are using it internally, so these new ones ought to as well.